submissions/onyemeMichael/html/product.html

```diff
@@ -78,7 +78,7 @@ <h5>Price: &#8358;<strong>88,000</strong></h5>
         <h3 class="kt">Sign up for our Newsletter</h3>
         <div class="newsletter">
             <form>
-                <input type="email" name="email" id="email" placeholder="Enter your email address" style="width: 50%; border-radius:0px; padding:5px">
+                <input type="email" name="email" id="let" placeholder="Enter your email address">
```

Comment:
Why are you changing the id of the email input field to `let`? @MYKEL7 I don't think the new `id` is the right name to use.